rt_utils/ds_helper.py

```diff
@@ -68,22 +68,22 @@ def add_study_and_series_information(ds: FileDataset, series_data):
     ds.SeriesDate = reference_ds.SeriesDate
     ds.StudyTime = reference_ds.StudyTime
     ds.SeriesTime = reference_ds.SeriesTime
-    ds.StudyDescription = reference_ds.StudyDescription
-    ds.SeriesDescription = reference_ds.SeriesDescription
+    ds.StudyDescription = getattr(reference_ds, 'StudyDescription', '')
+    ds.SeriesDescription = getattr(reference_ds, 'SeriesDescription', '')
     ds.StudyInstanceUID = reference_ds.StudyInstanceUID
-    ds.SeriesInstanceUID = generate_uid() # TODO: find out if random generation is ok
+    ds.SeriesInstanceUID = generate_uid()
     ds.StudyID = reference_ds.StudyID
     ds.SeriesNumber = "1" # TODO: find out if we can just use 1 (Should be fine since its a new series)
```

Comment:
Was this ever resolved?

Reply:
I think we concluded it should be fine given that a unique id is required here. Undid comment removal just in case